.core_files.yaml

```diff
@@ -126,9 +126,11 @@ tests: &tests
   - tests/*.py
   - tests/auth/**
   - tests/backports/**
+  - tests/components/diagnostics/**
```

Comment:
Added as similarly to repairs, changes to these function might break tests for other integrations